code/modules/mob/new_player/new_player.dm

```diff
@@ -398,6 +398,17 @@
 		if(job && IsJobAvailable(job.title))
 			available_job_count++;
 
+	if(length(SSjob.prioritized_jobs))
+		dat += "<div class='notice red'>The Head of Personnel's office has flagged these jobs as high priority:<br>"

```

Comment:
official stuff often is labeled as coming from "The Office Of Donklord Memefuck" or something